src/feedback/FeedbackDialog.css

```diff
@@ -36,8 +36,8 @@
 
 .dialog {
   position: fixed;
-  width: max-content;
-  padding: 16px;
+  width: 274px;
+  padding: 16px 22px 10px;
```

Comment:
Very specific sizes and paddings. Don't you think it would be wise to match out `4` based system `4, 8, 12, 16, 20, 24`?

Reply:
Good point, tweaked the width to `280` - I left the padding as is though, the `10` is because we apply `6px` elsewhere